fission/src/ui/panels/DebugPanel.tsx

```diff
@@ -84,15 +84,16 @@ const DebugPanel: React.FC<PanelPropsImpl> = ({ panelId }) => {
                     <LabelStyled>Autodesk Platform Services</LabelStyled>
                     <Button
                         value={"Refresh APS Token"}
-                        onClick={async () =>
-                            APS.isSignedIn() && APS.refreshAuthToken((await APS.getAuth())!.refresh_token, true)
-                        }
+                        onClick={async () => {
+                            const auth = await APS.getAuth()
+                            auth && APS.refreshAuthToken(auth.refresh_token, true)
+                        }}
```

Comment:
I think the syntax is a little messed up here. I believe since `auth` is a promise it always is evaluated as truthly. For this reason I think an if statement is required but correct me if I'm wrong.

Reply:
`auth` isn't a promise here, it's `APSAuth | undefined` because `APS.getAuth()` is `await`ed. It's a bit unconventional to use short-circuiting in JS like this (more common in react components) but this does work